### PR TITLE
Added huge page content to 4.0

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://jira.coreos.com/browse/OSDOCS-40

Preview build: http://file.rdu.redhat.com/~ahardin/04302019/huge-pages-4-0/scalability_and_performance/what-huge-pages-do-and-how-they-are-consumed-by-apps.html

https://jira.coreos.com/browse/ATS-190

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@jeremyeder @derekwaynecarr PTAL and let me know if this content is all still accurate for 4.0. Thanks!

**Comment by ahardin-rh** [comment]
@zvonkok PTAL

**Comment by ahardin-rh** [comment]
@zvonkok PTAL

**Comment by jeremyeder** [comment]
waiting for tuned section

**Comment by vikram-redhat** [comment]
@jeremyeder I am guessing this is still relevant for 4.0?

**Comment by jeremyeder** [comment]
yes, still relevant.

**Comment by ahardin-rh** [comment]
@jeremyeder @zvonkok Any updates?

**Comment by ahardin-rh** [comment]
@zvonkok Any updates?

**Comment by ahardin-rh** [comment]
@zvonkok @mffiedler PTAL. This is updated based off of the latest notes from Zvonko, including a  new "Configuring huge pages" section.

Updated preview build:
http://file.rdu.redhat.com/~ahardin/04302019/huge-pages-4-0/scalability_and_performance/what-huge-pages-do-and-how-they-are-consumed-by-apps.html

File: modules/how-huge-pages-are-consumed-by-apps.adoc [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  containers:
+  - securityContext:
+      privileged: true
+    image: rhel7:latest
+    command:
+    - sleep
+    - inf
+    name: example
+    volumeMounts:
+    - mountPath: /var/lib/hugepages
```
Comment (zvonkok) [inline modules/how-huge-pages-are-consumed-by-apps.adoc]:
```suggestion
    - mountPath: /dev/hugepages
```
The hugetlbfs on RHCOS is mounted on /dev/hugepages

Comment (ahardin-rh) [reply modules/how-huge-pages-are-consumed-by-apps.adoc]: Thanks! Updated

**Comment by zvonkok** [comment]
@ahardin-rh  Looks good to me.

**Comment by ahardin-rh** [comment]
@mffiedler PTAL

**Review by mffiedler — APPROVED** [review]

**Pull request merged**